addons/mail/static/src/core/common/composer.xml

```diff
@@ -2,24 +2,33 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.Composer">
+    <t t-set="isChatterComposer" t-value="extended and !props.composer.message"/>
+    <t t-set="partitionedActions" t-value="composerActions.partition"/>
     <t t-set="compact" t-value="props.mode === 'compact'"/>
     <t t-set="normal" t-value="props.mode === 'normal'"/>
     <t t-set="extended" t-value="props.mode === 'extended'"/>
-    <div t-ref="composer">
+    <t t-set="actionsContainerClass" t-value="'o-mail-Composer-actions d-flex rounded-3'"/>
```

Comment:
I have never been a fan of `t-set`, can't those be getters? It's easier if we need to override them in the future. 

Also, the `partionedActions` look useless?

Reply:
`partitionedActions` is to memoize the result from `composerActions.partition`

Reply:
`actionContainerClass` being a `t-set` has the advantage to be immediately visible in code of template, so easier to grep there. Getter is more powerful but it's annoying without single-file component...

So there's pros and cons with both approaches. Don't have strong stance either way although slight preference for `t-set` when this is enough.